x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_schema/model/rule_schemas.ts

```diff
@@ -104,6 +104,14 @@ export const RuleSourceCamelCased = z.discriminatedUnion('type', [
   z.object({
     type: z.literal('external'),
     isCustomized: IsExternalRuleCustomized,
+    customizedFields: z
```

Comment:
do we need to add new parameter to external schemas too?

Reply:
That's what [this PR](https://github.com/elastic/kibana/pull/234793) is doing. We need to add it to the internal rule schemas first due to our intermediate release practice with rule schema modifications